test/unittest/CMakeLists.txt

```diff
@@ -28,7 +28,7 @@ project(syclblas_unittest)
 
 set(SYCLBLAS_UNITTEST ${CMAKE_CURRENT_SOURCE_DIR})
 
-include_directories(${SYCLBLAS_TEST} ${BLAS_INCLUDE_DIRS})
+include_directories(${SYCLBLAS_TEST} ${BLAS_INCLUDE_DIRS} "${SYCLBLAS_TEST}/../external/clara")
```

Comment:
What is this doing here?

Reply:
You're the author, are you asking this question to yourself?

Reply:
Just making a note to myself.